mongo_orchestration/apps/__init__.py

```diff
@@ -45,7 +47,9 @@ def setup_versioned_routes(routes, version=None):
 def send_result(code, result=None):
     response.content_type = None
     if result is not None and 200 <= code < 300:
-        result = json.dumps(result)
+        # Use json.dumps instead of json_util.dumps to only encode
+        # non JSON serializable BSON types with json_util.
+        result = json.dumps(result, default=json_util.default)
```

Comment:
Making canonical the default would require a major version bump of PyMongo. We can cross that bridge when we get to it.

Instead of json.dumps(result, default=json_util.default) just use json_util.dumps, which properly handles types like Binary and Code.